spec/support/rspec_request_specs.rb

```diff
@@ -6,6 +6,12 @@ def app
   end
 end
 
+module CapybaraPage
+  def page
+    Capybara.string(response.body)
+  end
+end
```

Comment:
Side-effect of moving the token spec. This allows us to use Capybara selectors on request specs

Reply:
Nice one!